### PR TITLE
Improve style (It's not a test)

### PR DESCRIPTION
**Pull request by martinmoene**
Again, hope you like my changes...

I didn't fully understand *paper or word specifications* in the following phrase:
>From your experience, you may know *paper or word specifications* written in plain English or other spoken language. 

I read it as 'old-fashioned requirement specification', but I may well be wrong about that.
It's likely worth it to try and improve that sentence.

---
As an aside: I'm curious if you return to your earlier chapters now and then. In the process of writing your experience grows and your style may change somewhat. Re-reading what you've written previously you may discover things you'd like to change/improve.

### CODE REVIEW

File: manuscript/050_Its_Not_a_Test.md [hunk 1]
```diff
@@ -68,12 +68,12 @@ The time has come to make a deal: I will establish some naming conventions for
...
+From your experience, you may know paper or word specifications written in plain English or other spoken language. Our specification is different from these specifications in at least few ways:
 
-1.  it is not written fully up-front (more on this in the next chapters).
-2.  it is executable -- you can run it to see whether the code adheres to the specification or not.
-3.  it is written in source code rather than in spoken language -- which is both good (less room for misunderstanding -- source code is the most formal and structured way of writing specifications) and bad (great care must be taken to keep the specification readable).
+1.  It is not completely written up-front (more on this in the next chapters).
+2.  It is executable -- you can run it to see whether the code adheres to the specification or not.
+3.  It is written in source code rather than in spoken language -- which is both good, as there is less room for misunderstanding and bad, as great care must be taken to keep the specification readable.
```
Comment (grzesiek-galezowski) [inline manuscript/050_Its_Not_a_Test.md]: I like the re-structured sentence, however, I'd like to keep the information somehow that the reason source code leaves less room for understanding is that it is less ambiguous compared to spoken language.

File: manuscript/050_Its_Not_a_Test.md [hunk 2]
```diff
@@ -68,12 +68,12 @@ The time has come to make a deal: I will establish some naming conventions for
 
 :   will be used instead of ‘passing test’
 
-From time to time, I will refer back to the ‘traditional’ terminology, because it is better established and you’ve probably heard some terms already and may wonder how it should be understood in context of thinking of tests as specification.
+From time to time I'll refer back to the ‘traditional’ terminology, because it is better established and because you’ve probably already heard some terms and may wonder how it should be understood in the context of thinking of tests as specification.
 
-From your experience, you may know paper or word specifications, written in plain English or other spoken language. Our specification is different than these specifications in at least few ways:
+From your experience, you may know paper or word specifications written in plain English or other spoken language. Our specification is different from these specifications in at least few ways:
```
Comment (grzesiek-galezowski) [inline manuscript/050_Its_Not_a_Test.md]: You are right that by this sentence, I meant old-fashioned specifications, though I think "old-fashioned" is a bit ambiguous. I really meant requirements or design specifications that were usually considered a precondition to start implementing something.

**Comment by grzesiek-galezowski** [comment]
By the way, I sometimes come back to old chapters, as even my views sometimes change a bit. Still, there is always a trade-off between adding new content and polishing old. For now, I put a little more emphasis on adding new content. This is another reason why your help with the existing content is so valuable :-). I have some plans, e.g. looking again at the triangulation chapter as I had a chance to practice it more in recent months.

The plan is for the book to "never be finished" - I like how this approach works in computer games (I mean the "game as a service" idea) - the product is continuously improved thanks to feedback from the community, technology advancement and improved skills of the authors. This is just a concept though :-)

**Comment by martinmoene** [comment]
Thanks for your remarks. Hope to continue on this tomorrow.

**Comment by martinmoene** [comment]
Tried to incorporate your feedback so far.

**Comment by grzesiek-galezowski** [comment]
Thanks for including the feedback!
